instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2SqsTracingTest.groovy

```diff
@@ -177,4 +169,38 @@ abstract class AbstractAws2SqsTracingTest extends InstrumentationSpecification {
       }
     }
   }
+
+  def "simple sqs producer-consumer services: sync"() {
+    setup:
+    def builder = SqsClient.builder()
```

Comment:
usually we parametrise such tests https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/ca24c388463cb36baa4b3926b26d7fc4b348b237/instrumentation/jaxws/jaxws-2.0-common-testing/src/main/groovy/AbstractJaxWsTest.groovy#L121 could also use true/false or even the builder itself as the parameter like in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/ca24c388463cb36baa4b3926b26d7fc4b348b237/instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2ClientTest.groovy#L379

Reply:
I did not do this because there are different types involved with no common interface type, even though the method names are the same. I guess in groovy due to the dynamic typing it would not technically matter, but it might impact auto-completion (and I'd have to add `if (is_async) { result.get() }` or similar to wait on the futures conditionally)

Reply:
makes sense, didn't notice the `.get()` calls